tests/plugin_tests/worker_api.cpp

```diff
@@ -17,6 +19,260 @@ struct worker_api_fixture : public golos::chain::database_fixture {
 
 BOOST_FIXTURE_TEST_SUITE(worker_api_plugin, worker_api_fixture)
 
-// TODO: Start write tests in #1025 issue
+BOOST_AUTO_TEST_CASE(worker_proposal_validate) {
+    BOOST_TEST_MESSAGE("Testing: worker_proposal_validate");
+
+    BOOST_TEST_MESSAGE("-- Default type case");
+
+    worker_proposal_operation op;
+    op.author = "alice";
+    op.permlink = "test";
+    BOOST_CHECK_NO_THROW(op.validate());
+    BOOST_CHECK(op.type == worker_proposal_type::task);
```

Comment:
Default parameters doesn't work in the Graphene, because raw packing doesn't allow them.
You should set all paramaters by yourself.